packages/metal-dom/test/events.js

```diff
@@ -102,4 +109,86 @@ describe('Custom Events', function() {
 		assert.strictEqual(1, listener.callCount);
 		assert.strictEqual('animationend', listener.args[0][0].customType);
 	});
+
+	it('should listen to the keydown keyboard event by keyCode alias', function() {
+		var element = document.createElement('input');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keydown:enter', listener);
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.ENTER});
+		assert.strictEqual(1, listener.callCount);
+	});
+
+	it('should delegate parameterized keyboard event', function() {
+		var listener = sinon.stub();
+		var handle = dom.delegate(this.element4, 'keydown:enter', this.element5, listener);
```

Comment:
Couldn't this test use `element2` and `element3` instead of creating two new elements? That would simplify things.